Lib9c/Action/InitializeStates.cs

```diff
@@ -29,27 +29,27 @@ namespace Nekoyume.Action
     [ActionType("initialize_states")]
     public class InitializeStates : GameAction
     {
-        public Bencodex.Types.Dictionary Ranking { get; set; } = Dictionary.Empty;
-        public Bencodex.Types.Dictionary Shop { get; set; } = Dictionary.Empty;
+        public Dictionary Ranking { get; set; } = Dictionary.Empty;
+        public Dictionary Shop { get; set; } = Dictionary.Empty;
         public Dictionary<string, string> TableSheets { get; set; }
-        public Bencodex.Types.Dictionary GameConfig { get; set; } = Dictionary.Empty;
-        public Bencodex.Types.Dictionary RedeemCode { get; set; } = Dictionary.Empty;
+        public Dictionary GameConfig { get; set; } = Dictionary.Empty;
+        public Dictionary RedeemCode { get; set; } = Dictionary.Empty;
 
-        public Bencodex.Types.Dictionary AdminAddress { get; set; } = Dictionary.Empty;
+        public Dictionary AdminAddressState { get; set; } = Dictionary.Empty;
```

Comment:
I thought it's typo issue by seeing [here](https://github.com/planetarium/lib9c/blob/96a122732b6e788e342f47eeb5efeb5048beec0f/Lib9c/Action/InitializeStates.cs#L77); "Why serialize result of `adminAddressState` named `adminAddress`?" 
Am I missing something?

Reply:
Ah, I didn't intend to remove this part. This comment was for splitting the commit 😅 Because the commit was for '[Remove redundant qualifier](https://github.com/planetarium/lib9c/pull/1292/commits/ee7b6e06a3e31a2cedf973f71f0aca31901fe19c)' but it also renamed the variable unexpectedly. That's what I wanted to say.

Reply:
@moreal  Oh, I see. As you say, it's my bad using `git add -p` command.  sry for mixed commit contents.